specifications/sep-payment-channel-transactions.md

```diff
@@ -319,9 +319,14 @@ change `minSeqAge` and `minSeqLedgerGap` to zero.
 
 1. Submit most recent D_i
 2. Modify the most recent C_i `minSeqAge` and `minSeqLedgerGap` to zero
-3. Resign and exchange the modified confirmation transaction C_i
+3. Resign and exchange the modified close transaction C_i
 4. Submit modified C_i
 
+If participants choose to coordinate a close before submitting D_i they must
+take care to also modify D_i such that it contains the new C_i's hash is
+included in the new D_i's `extraSigners` or to keep a copy of the old C_i's
+signatures required to satisfy D_i's `extraSigners` precondition.
```

Comment:
@acharb I added this warning to the SEP following our conversation.

Reply:
👍 . I think there's a typo though, should be?:
```
If participants choose to coordinate a close before submitting D_i they must
take care that the new C_i's hash is included in the D_i's `extraSigners`,
or keep a copy of the old C_i's signatures required to satisfy D_i's `extraSigners` 
precondition.
```

Reply:
```suggestion
If participants choose to coordinate a close before submitting D_i they must
take care that the new C_i's hash is included in the D_i's `extraSigners`,
or keep a copy of the old C_i's signature required to satisfy D_i's `extraSigners` 
precondition.
```